config/default-ocp/kustomization.yaml

```diff
@@ -0,0 +1,44 @@
+# Adds namespace to all resources.
+namespace: kueue-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kueue-
+
+labels:
+  - pairs:
+      app.kubernetes.io/name: kueue
+      app.kubernetes.io/component: controller
+    includeTemplates: true
+  - pairs:
+      control-plane: controller-manager
+    includeSelectors: true
+
+resources:
+- ../components/crd
+- ../components/rbac
+- ../components/manager-ocp
+- ../components/visibility
+- ../components/webhook
+- ../components/certmanager
+# [METRICS] Expose the controller manager metrics service.
+- metrics_service.yaml
+
+transformers:
+# Sets the namespace for the role binding as kube-system instead of default kueue-system
+- role_binding_visibility_transformer.yaml
+
+patches:
+# Mount the controller config file for loading manager configurations
+# through a ComponentConfig type
+- path: manager_config_patch.yaml
```

Comment:
I think you need to include the webhook patches here for validating and mutating.